.travis.yml

```diff
@@ -175,9 +169,6 @@ matrix:
     - python: "3.7"
       env: TOXENV=py37
       <<: *extended-test-suite
-    - python: "3.8"
```

Comment:
Isn't it 3.7 that you want to remove?

Reply:
No.

I should have probably made it clearer in my comment, but instead I left the Python 3.8 tests in the `not-on-master` tests at https://github.com/certbot/certbot/pull/7685/files#diff-354f30a63fb0907d4ad57269548329e3R65.

Reply:
OK, got it!